### PR TITLE
Staff Pay Timing Issue and Mhr Table UX

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#13260

*Description of changes:*
* UX updates as described in ticket
* Fixed Filing Submission Timing issue

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Review by ChasBoucher — APPROVED** [review]
Looks Good

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 1]
```diff
@@ -406,8 +406,7 @@ export default defineComponent({
 
     const downloadPDF = async (item: RegistrationSummaryIF): Promise<any> => {
       localState.loadingPDF = item.path
-      // const pdf = await registrationPDF(item.path)
-      const pdf = await mhRegistrationPDF(item.path)
+      const pdf = props.isPpr ? await registrationPDF(item.path) : await mhRegistrationPDF(item.path)
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/common/TableRow.vue:409]: @doug-lovett 
My blunder. 
Fixed here!

Comment (doug-lovett) [reply ppr-ui/src/components/tables/common/TableRow.vue:409]: Great, nice when it's so straightforward.

Comment (cameron-eyds) [reply ppr-ui/src/components/tables/common/TableRow.vue:409]: Indeed!

**Pull request merged**